Makefile

```
@@ -602,12 +602,20 @@ docker-build-all: $(addprefix docker-build-,$(ALL_ARCH)) ## Build docker images
 docker-build-%:
 	$(MAKE) ARCH=$* docker-build
 
-ALL_DOCKER_BUILD = core kubeadm-bootstrap kubeadm-control-plane clusterctl
+ALL_DOCKER_BUILD = core kubeadm-bootstrap kubeadm-control-plane capd test-extension clusterctl
```

```suggestion
ALL_DOCKER_BUILD = core kubeadm-bootstrap kubeadm-control-plane docker-infrastructure test-extension clusterctl
```
WDYT? For consistent naming with other providers (+ other targets similarly)

---

done